### ISSUE TITLE
Getters and setters.

### ISSUE THREAD

**Issue by NeilFraser** [JC-Orozco/BlocksIDE#16]
var o = {
  get property() {return 4},
  set property(value) {alert(value)}
};

**Comment by JC-Orozco** [comment]
This will require some new blocks. Specifically set and get expression blocks to be able to plug them to maps

**Comment by JC-Orozco** [comment]
Expressions must also be allowed:
```
var expr = 'foo';

var obj = {
  baz: 'bar',
  set [expr](v) { this.baz = v; }
};
```

**Comment by JC-Orozco** [comment]
Fixed original bug case on Version 0.3.17.
I will add expression property case to another bug report.

**Comment by JC-Orozco** [comment]
Fixed second example on version 0.3.18
